src/guiguts/application.py

```diff
@@ -353,6 +353,33 @@ def init_edit_menu(self) -> None:
         menu_edit.add_button(
             "Colu~mn Paste", maintext().columnize_paste, "Cmd/Ctrl+Shift+V"
         )
+        menu_edit.add_separator()
+        # NOTE: this was "~l" in GG1; but "~l" is now used by Column Cut, using "~w"
+        menu_edit.add_button(
+            "lo~wercase selection",
+            lambda *args: maintext().transform_selection(str.lower),
+            "Ctrl+l",
+        )
+        menu_edit.add_button(
+            "~Sentence case selection",
+            lambda *args: maintext().transform_selection(
+                maintext().sentence_case_transformer
+            ),
+            "",
+        )
+        menu_edit.add_button(
+            "~Title Case Selection",
+            lambda *args: maintext().transform_selection(
+                maintext().title_case_transformer
+            ),
+            "Ctrl+t",
+        )
+        # NOTE: this was "~u" in GG1; but "~u" is now used by Undo, using "~p"
```

Comment:
I haven't particularly been trying to preserve the GG1 "~" letters, so I'm fine with this.

Reply:
I realized as I was falling asleep last night that I left this counter to my own hotkey advice, so I'll look at this. Maybe I'll do the platform one... or maybe this should grow a shift key or something...

Reply:
> I haven't particularly been trying to preserve the GG1 "~" letters, so I'm fine with this.

Just to clarify, what I read this to mean is: you are fine with the `~` letter changing, but each menu item still ought to *have* one. Is that right?

By the way, I have not been testing those, and I don't think I can. The menu behavior is different in macOS. When a menu is open, you can press letters to jump to items, but it's based on the text label, not on the `~` letters. Example: in the current Edit menu, I can press `c o l` and it will jump as I type to `Cut` (c), then `Copy` (co), and finally `Column Cut` (col). In GG1, I could use the `~` letter because the menu handling was done by XQuartz, not macOS. This is all fine and normal, it's just information for you in case you weren't already aware.

Also for developer notes: if two menu items have the same `~` letter, Python doesn't complain about it (at least, on macOS it doesn't - maybe it does on other platforms?)

Reply:
Sorry - on re-reading it, you're right, I wasn't clear. I am fine with the letter changing, but yes, let's keep a `~` letter for every menu item. If at any point in the future we decide not try not supporting them, it should be a simple fix in `process_label` to 
`    return (-1, label.replace("~", ""))`

I didn't know (or remember?) that they weren't supported on macOS - thanks for that info. As you described, I see in [TkDocs](https://tkdocs.com/tutorial/menus.html#:~:text=Underline) it says (my emphasis) 

> All platforms support keyboard traversal of the menubar via the arrow keys. On **_Windows and X11_**, you can also use other keys to jump to particular menus or menu items. The keys that trigger these jumps are indicated by an underlined letter in the menu item's label. 

One thing that may cause complaints from GG1 users is that (on the basis of stuff I read about modernising a Tk interface) I have disabled the "tear-off" functionality for menus in GG2, though it's entirely possible we could have that as a user preference - I confess I used it a lot in GG1. So, without tear-offs, using `Alt+E`, then `T` provides quick keyboard access to Title Case, for example, even if we don't have the Ctrl+t shortcut. On Macs, it sounds like a few arrow keys in addition would be needed. 

On Windows, if two menu items have the same `~` letter, the first one in the menu is selected by the keystroke.

Reply:
@windymilla I'm curious if, in macOS, the tear-off menus are even possible. You say that you disabled the feature--can you point me to where that was? I'd like to try enabling it, to see what happens.

Reply:
> Are you happy that this will make `Ctrl+t` do Title Case on Macs? I know you were concerned previously about not overwriting shortcut Control keys unnecessarily.

I removed (for Mac only) the hotkeys for these menu items. We can always adjust later, but (as discussed in Slack) I'm thinking in the direction of hotkeys only for the more commonly used commands.

Reply:
AFK, but I think in Root class init constructor in Mainwindow. I think it is called tearoff or tear-off

Reply:
Agreed, let's keep to a minimum for now, then add them deliberately for common commands, maybe after some consultation with users. I'm happy for you to remove for other users too.

Reply:
Back at computer, @tangledhelix, the tearOff setting is in the Root constructor (about line 46)
```python
        self.option_add("*tearOff", False)
```

Reply:
@windymilla As I suspected, tearOff does nothing on macOS. Thanks for the pointer.

Reply:
> I'm happy for you to remove for other users too.

Removed entirely (not only for Mac)